lib/active_storage_validations/aspect_ratio_validator.rb

```diff
@@ -73,18 +73,18 @@ def is_valid?(record, attribute, metadata)
         return true if metadata[:width] > metadata[:height]
         add_error(record, attribute, :aspect_ratio_not_landscape)
 
-      else
-        if options[:with] =~ /is\_(\d*)\_(\d*)/
-          x = $1.to_i
-          y = $2.to_i
+      when /is_(\d*)_(\d*)/

```

Comment:
These should probably be `(\d+)` as `is__1` or `is_5_` would be currently allowed.

Reply:
```ruby
when /\Ais_(\d*[1-9]\d*)_(\d*[1-9]\d*)\z/
```
would also ensure the string is exactly `is_X_Y` with X and Y being > 0.

Reply:
I used `/is_([1-9]\d*)_([1-9]\d*)/` which is, I think, a bit better since it will not accept ratios like `is_01_2`